pythonforandroid/bootstraps/common/build/src/main/java/org/kivy/android/DashCamPythonService.java

```diff
@@ -0,0 +1,212 @@
+package org.kivy.android;
+
+import android.os.Build;
+import java.lang.reflect.Method;
+import java.lang.reflect.InvocationTargetException;
+import android.app.Service;
+import android.os.IBinder;
+import android.os.Bundle;
+import android.content.Intent;
+import android.content.Context;
+import android.util.Log;
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.os.Process;
+import java.io.File;
+
+//imports for channel definition
+import android.app.NotificationManager;
+import android.app.NotificationChannel;
+import android.graphics.Color;
+
+public class DashCamPythonService extends Service implements Runnable {
+
+    private static final String SERVICE_ENTRY_POINT= "gen3cam/services/dashCamUploaderService.py";
```

Comment:
This is the reason for example why this code should be part of the python repo and not in this repo. Direct dependency to an outside project.

Reply:
I share the same view. Refer above comment

Reply:
Not an expert, but could this be a smaller solution:
https://developer.android.com/reference/android/app/Service#START_REDELIVER_INTENT
Not sure how it behave with intents delivered in the meantime and when exactly in intent is delivered.